x/wasm/keeper/keeper.go

```diff
@@ -596,6 +597,25 @@ func (k Keeper) removeFromContractCodeSecondaryIndex(ctx sdk.Context, contractAd
 	ctx.KVStore(k.storeKey).Delete(types.GetContractByCreatedSecondaryIndexKey(contractAddress, entry))
 }
 
+// addToContractCreatorSecondaryIndex adds element to the index for contracts-by-creator queries
+func (k Keeper) addToContractCreatorSecondaryIndex(ctx sdk.Context, creatorAddress, contractAddress sdk.AccAddress) {
+	store := ctx.KVStore(k.storeKey)
+	bz := address.MustLengthPrefix(creatorAddress)
```

Comment:
This should move into the `GetContractByCreatorSecondaryIndexKey` method so that full key is built in that function. It is easy to miss otherwise.